client/components/data/query-domain-nameservers/README.md

```diff
@@ -0,0 +1,35 @@
+# QueryDomainNameservers
+
+`<QueryDomainNameservers />` is a React component used in managing network requests for domain nameservers for a given domain name.
+
+## Usage
+
+Render the component, passing `domainName`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import QueryDomainNameservers from 'calypso/components/data/query-domain-nameservers';
+
+export default function MyDomainNameservers( { domainName, nameservers } ) {
+	return (
+		<div>
+			<QueryDomainNameservers domainName={ domainName } />
+
+			{ nameservers.list?.map( ( nameserver ) => (
```

Comment:
There's a hint in the _Usage_ part of this doc but do we have to extend the example so that `MyDomainNameservers` is actually a `connect()`ed component?

Reply:
Good call, expanded the example in 2f91c69be65776f4c29fd55bd0ef9d30054526a4